docs/configure

```diff
@@ -1025,6 +1025,75 @@ login_virtual_exim:
         {*}{}}
 
 
+# If you use more advanced password hashing (sha512 or bcrypt), exim cannot handle
```

Comment:
If there are cases where Exim CAN handle these hashes itself, here we should say that it "may not be able to handle" instead of "cannot handle" them.